apps/server/src/modules/common-cartridge/service/common-cartridge-import.service.ts

```diff
@@ -47,14 +74,124 @@ export class CommonCartridgeImportService {
+			const cardsWithoutResource = cards.filter((card) => !card.isResource);
+
+			await this.columnClient.updateBoardColumnTitle(columnResponse.id, { title: column.title });
+
+			for await (const card of cardsWithResource) {
+				await this.createCardWithElement(parser, columnResponse, card);
+			}
+
+			for await (const card of cardsWithoutResource) {
+				await this.createCard(parser, columnResponse, card);
+			}
+		}
+	}
+
+	private async createCard(
+		parser: CommonCartridgeFileParser,
+		column: ColumnResponse,
+		cardProps: CommonCartridgeImportOrganizationProps
+	): Promise<void> {
+		const organizations = parser.getOrganizations();
+		const cardElements = organizations.filter(
+			(organization) => organization.pathDepth >= DEPTH_CARD_ELEMENTS && organization.path.startsWith(cardProps.path)
+		);
+
+		const card = await this.columnClient.createCard(column.id, {});
+
+		for await (const cardElement of cardElements) {
+			await this.createCardElement(parser, card.id, cardElement);
+		}
+	}
+
+	private async createCardWithElement(
+		parser: CommonCartridgeFileParser,
+		column: ColumnResponse,
+		cardProps: CommonCartridgeImportOrganizationProps
+	): Promise<void> {
+		const card = await this.columnClient.createCard(column.id, {});
+
+		await this.cardClient.updateCardTitle(card.id, {
+			title: cardProps.title,
+		});
+
+		await this.createCardElement(parser, card.id, cardProps);
+	}
+
+	private async createCardElement(
+		parser: CommonCartridgeFileParser,
+		cardId: string,
+		cardElementProps: CommonCartridgeImportOrganizationProps
+	): Promise<void> {
+		if (cardElementProps.isResource) {
+			const resource = parser.getResource(cardElementProps);
+			const contentElementType = CommonCartridgeImportMapper.mapResourceTypeToContentElementType(resource?.type);
+
+			if (resource && contentElementType) {
+				const resourceBody = this.mapToResourceBody(resource, cardElementProps);
+				const contentElement = await this.cardClient.createCardElement(cardId, { type: contentElementType });
+				if (resourceBody) {
+					await this.cardClient.updateCardElement(contentElement.id, { data: resourceBody });
+				}
+			}
+		}
+	}
+
+	private mapToResourceBody(
+		resource: CommonCartridgeImportResourceProps,
+		cardElementProps: CommonCartridgeImportOrganizationProps
+	): UpdateElementContentBodyParamsData | undefined {
+		if (resource.type === CommonCartridgeResourceTypeV1P1.WEB_LINK) {
+			return this.createLinkFromResource(resource);
 		}
+
+		if (
+			resource.type === CommonCartridgeResourceTypeV1P1.WEB_CONTENT &&
+			cardElementProps.resourcePath.endsWith('.html')
+		) {
+			return this.createTextFromHtmlResource(resource);
+		}
+
+		return undefined;
+	}
+
+	private createTextFromHtmlResource(
+		resource: CommonCartridgeImportWebContentResourceProps
+	): RichTextElementContentBody {
+		return {
+			type: 'richText',
+			content: {
+				inputFormat: InputFormat.RICH_TEXT_CK4, // TODO use config
+				text: resource.html,
+			},
+		};

```

Comment:
```suggestion
		const richTextBody: RichTextElementContentBody = {
        type: 'richText',
        content: {
            inputFormat: InputFormat.RICH_TEXT_CK4, // TODO use config
            text: resource.html,
        },
    };

    return richTextBody;
```
I think this is easier to debug.